helper/schema/resource_data.go

```diff
@@ -213,9 +212,11 @@ func (d *ResourceData) State() *terraform.InstanceState {
 		}
 
 		raw := d.get([]string{k}, source)
-		rawMap[k] = raw.Value
-		if raw.ValueProcessed != nil {
-			rawMap[k] = raw.ValueProcessed
+		if raw.Exists && !raw.Computed {
```

Comment:
Check, that's clear! So indeed the ACC test seems to hit a few small, but related things at once... Good ACC test :wink: